CONTRIBUTING.md

```diff
@@ -452,13 +452,13 @@ The Asciidoc files can be found in the [`src/main/asciidoc` directory](https://g
 
 When contributing a significant documentation change, it is highly recommended to run the build and check the output.
 
-First build the whole Quarkus repository with the documentation build enabled (`-Dquickly` skips the documentation build):
+First build the whole Quarkus repository with the documentation build enabled:
 
 ```
-./mvnw -Dquickly -DskipDocs=false clean install
+./mvnw -DskipTests -DskipITs -DskipExtensionValidation -Dskip.gradle.tests clean install
```

Comment:
Could we maybe create another profile that would be activated by `-DquicklyDocs`? That would be easier.